### ISSUE TITLE
Big UI change in Atom 1.17.0-beta0

### ISSUE THREAD

**Issue by pingshunhuangalex** [atom-material/atom-material-ui#376]
The coming `Atom 1.17.0-beta0` introduced a lot of things in UI which breaks the theme in many aspects. The most noticeable one is the `panel contrasting` stops working and there is an obvious black line between tree view and workspace in tab section.

![image](https://cloud.githubusercontent.com/assets/17170199/24987815/42ca7f16-2045-11e7-9d59-b58fe17ae239.png)

The latest version also introduced the concept of dock where the theme may need to adapt too.

**Comment by silvestreh** [comment]
Thanks for the heads up! I'm targeting Atom 1.17 in the `2.0` branch… I'll pull the latest changes from Atom's repo and rebuild 👍

**Comment by simonepri** [comment]
Also the scroll of the tree-view does not work anymore in `Atom 1.17.0-beta0`

**Comment by silvestreh** [comment]
@simonepri what do you mean by _the scroll_?

**Comment by pingshunhuangalex** [comment]
@silvestreh 
It means if the content is longer than the height of the tree view, there will be no scroll bar shown, nor does it respond to mouse wheel for scrolling.

Can confirm this on my end too, and it works on Atom's default theme, so it should be related to `atom-material-ui`

**Comment by silvestreh** [comment]
Oh, it's absolutely broken then! That's odd because I don't remember messing with the scroll behavior… Anyway, it seems to be working just fine on the `2.0` branch.

**Comment by pingshunhuangalex** [comment]
@silvestreh 
I'm really looking forward to 2.0, so I'll just need to wait for the official 1.17.0 to come right?
I heard `material-ui` will be compatible with `nuclide` in 2.0. Am I right?

**Comment by silvestreh** [comment]
@pingshunhuangalex yeah! 2.0 will be 99% less awkward with all packages. 😄

**Comment by simonepri** [comment]
@silvestreh Could you publish on `apm` a **beta** version of the `2.0` branch ?

**Comment by silvestreh** [comment]
@simonepri I don't think I can publish beta versions to APM.

**Comment by silvestreh** [comment]
This is fixed in 736a966bdce11108c24f38d78bd10900da51ac09. I'll leave the issue open for visibility while 2.0 remains unreleased.

**Comment by Evpok** [comment]
Any ETA for 2.0 ?

**Comment by silvestreh** [comment]
@Evpok as soon as Atom 1.17 becomes the latest stable

**Comment by aemino** [comment]
Suggestion for people who don't want to disable the theme while they wait... You can hackishly install 2.0 by cloning the repository into `.atom/packages`, using `git checkout 2.0` in the repository directory, and then relaunching Atom.

However, this has some... strange side effects.
![image](https://cloud.githubusercontent.com/assets/13167687/25048981/36e1cb14-20f4-11e7-99c2-40ef665979ea.png)

**Comment by rwatts3** [comment]
btw 2.0 is absolutely amazing, very happy to see this in development.

**Comment by Evpok** [comment]
@silvestreh Could you make a pre-release for 1.17-beta users ?

**Comment by silvestreh** [comment]
@Evpok you can clone the repo and link it as a package. 

```shell
git clone git@github.com:atom-material/atom-material-ui.git
cd atom-material-ui
git fetch origin
git checkout 2.0
apm link
```

**Comment by alexbuzzbee** [comment]
@silvestreh Didn't work with SSH; seems to over HTTPS. Odd.

**Comment by silvestreh** [comment]
`2.0` is available for Atom 1.17-beta users. Upgrade and flood the issue tracker! 😆
